src/lib/features/playground/playground-service.ts

```diff
@@ -73,14 +70,14 @@ export class PlaygroundService {
         );
         const items = results.flat();
         const itemsByName = groupBy(items, (item) => item.name);
-        return Object.entries(itemsByName).map(([name, entries]) => {
+        return Object.values(itemsByName).map((entries) => {
             const groupedEnvironments = groupBy(
                 entries,
                 (entry) => entry.environment,
             );
             return {
-                name,
-                projectId: entries[0]?.projectId,
+                name: entries[0].name,
```

Comment:
each group is guaranteed to have at least one element because of how groupBy works